tests/test_infra.py

```diff
@@ -62,7 +62,7 @@ def test_ensure_copyright():
 def test_setup_version():
     """Verify that setup.py is picking up the version correctly."""
     cmd = [os.path.abspath("setup.py"), "--version"]
-    proc = subprocess.run(cmd, stdout=subprocess.PIPE)
+    proc = subprocess.run(cmd, stdout=subprocess.PIPE, check=True)
```

Comment:
The subprocess.run changes were manual, just adding `check=True` when there was no check parameter.